api/cluster.go

```diff
@@ -52,7 +53,7 @@ func (s *Server) indexFind(ctx *middleware.Context, req models.IndexFind) {
 	for _, pattern := range req.Patterns {
 		nodes, err := s.MetricIndex.Find(req.OrgId, pattern, req.From)
 		if err != nil {
-			response.Write(ctx, response.NewError(http.StatusBadRequest, err))
+			response.Write(ctx, response.NewError(http.StatusBadRequest, err.Error()))
```

Comment:
it definitely is needed.  Your example code is broken. fix it and you get the error.
```
cannot use err (type error) as type string in argument to printStr
```